hashes/contrib/test.sh

```diff
@@ -17,6 +17,12 @@ if cargo --version | grep nightly >/dev/null; then
     NIGHTLY=true
 fi
 
+# Pin dependencies as required if we are using MSRV toolchain.
+if cargo --version | grep "1\.48"; then
```

Comment:
Some sort of version comparison would be nice. `sort` can compare versions, so this hack should work:

```bash
if "$(echo -e "$(cargo | cut -d ' ' -f 2)"'\n1.48.0' | sort -V | tail -n 1)" = "1.48.0"; then
```

Reply:
I agree, but I don't think it belongs in this PR -- in a future thing we should maybe try to rewrite this script entirely to clean it up (though I think it's mostly in good shape) and to make it easier/more intuitive to run locally.

Reply:
Yeah, I didn't ACK because of the attribute.